src/main/java/cceclipseplugin/core/CCIgnore.java

```diff
@@ -141,7 +151,7 @@ private void serverCleanUp(IProject p) {
 			for (String ignored : ignoredFiles) {
 				if (path.startsWith(ignored) || path.equals(ignored)) {
 					// send delete request for fileID
-					System.out.println(String.format("Cleaning up %s from server", path));
+					Logger.getInstance().log(IStatus.INFO, String.format("Cleaning up %s from server", path));
```

Comment:
Also that logger variable was from a previous attempt to directly access the plugin's logger rather than through a wrapper class